app/models/network/node.rb

```diff
@@ -199,6 +199,11 @@ def units
       get(:units) || 1
     end
 
+    # Public: A string that returns the stakeholder
+    def stakeholder
```

Comment:
This method is the only thing I don't like about this PR – and it's a very minor thing, I know. 😆 

The stakeholder isn't an attribute relevant to the network calculation – i.e., the calculation of loads – so I don't think it belongs here. It's not guaranteed that this attribute will always be available (for example, if the `Network` module is extracted out to a separate gem), so I'm reluctant to add it as a first-class method on `Node`.

Can I suggest just using `get(:stakeholder)` for now, until I decide what I'm going to do with the `Network` module in the longer-term?